boot/bootutil/zephyr/CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
 # Copyright (c) 2020 Nordic Semiconductor ASA
+# Copyright (c) 2022 Legrand North America, LLC.
 #
 # SPDX-License-Identifier: Apache-2.0
 
 if(CONFIG_MCUBOOT_BOOTUTIL_LIB)
 
 zephyr_interface_library_named(MCUBOOT_BOOTUTIL)
 
-target_include_directories(MCUBOOT_BOOTUTIL INTERFACE
+zephyr_include_directories(MCUBOOT_BOOTUTIL INTERFACE
```

Comment:
why this change?

Reply:
Because the repartitioned build system couldn't see this otherwise, and this is a zephyr-specific build file.